# Projects website
### Computational Biology | Bioinformatics | Data Science
See https://akrammendez.github.io/

## Education
* Ph.D., Biochemistry, [Computational Biology Lab, Institute for Biomedical Research, UNAM](https://www.linkedin.com/in/dr-luis-mendoza)
* M.Sc., Boichemistry, [Computational Biology Lab, Institute for Biomedical Research, UNAM](https://www.linkedin.com/in/dr-luis-mendoza)
* Bachelor in Genomic Sciences, [Center for Genomic Sciences, UNAM](https://www.ccg.unam.mx/en/about/)

## Experience
* Postdoctoral Research Assistant, [Cancer Systems Biology, Francis Crick Institute](https://www.crick.ac.uk/research/labs/francesca-ciccarelli)
* Postdoctoral Researcher, [Epigenetics, Institute of Biomedicine, University of Gothenburg](https://www.gu.se/en/research/tanmoy-mondal)
* Bioinformatician, [Bioinformatics, Codigo46, R&D](http://linkedin.com/company/codigo46/)
* PhD fellow, [Computational Biology, Institute for Biomedical Research, UNAM](https://www.linkedin.com/in/dr-luis-mendoza)

## Projects
* Context-dependent effects of CDKN2A and other 9p21 gene losses during the evolution of oesophageal cancer.
[Publication](https://www.biorxiv.org/content/10.1101/2024.01.24.576991v1)
   - Analysis of next-generation sequencing (NGS) data in Oesophageal cancer.
   - Multiplexed imaging mass cytometry (IMC) Single-cell analysis.
   - Generated reports and collaborated in writing publications.

* Transcriptomics and Epigenetics in Neuroblastoma:
    [METTL3 drives telomere targeting of TERRA lncRNA through m6A-dependent R-loop formation: a therapeutic target for ALT-positive neuroblastoma.](https://doi.org/10.1093/nar/gkad1242)
    - Developed workflows to analyse epigenomics and transcriptomics data in Neuroblastoma: MeRIP-seq, ChIP-seq, bulk and single-cell RNA-seq and Nanopore long-read sequencing.
    - Coding using R, Shell, and other programming languages. Implemented workflows in cloud computing and HPC environments: AWS, SLURM.
    - Good coding practices and version control using GitHub.

* Bioinformatician at Codigo46: 
    * Harnessing large-scale genetic data for polygenic risk score disease prediction, pharmacogenomics, and population genetics.
![Analysis of genotypic data at Codigo46](/images/codigo46_workflow.png)

* Dynamical modeling of gene regulatory networks controlling Lymphocyte differentiation
    - Developed bioinformatic analysis workflows for genetic testing and Biobanking platforms: 
        Illumina GSA genotyping array, genome-wide association studies (GWAS, PRS), and human population genetics.
    - Led the data mining and annotation team.
    - Conducted statistical analysis, visualisation and interpretation to communicate with stakeholders.
    - Experience working with agile development methodologies (SCRUM).
![Dynamical modeling of gene regulatory networks controlling Lymphocyte differentiation](/images/network_workflow.png)