# Projects website
### Computational Biology | Bioinformatics | Data Science

[[toc]]

## Education {#education}
* Ph.D., Biochemistry, [Computational Biology Lab, Institute for Biomedical Research, UNAM](https://www.linkedin.com/in/dr-luis-mendoza)
* M.Sc., Boichemistry, [Computational Biology Lab, Institute for Biomedical Research, UNAM](https://www.linkedin.com/in/dr-luis-mendoza)
* Bachelor in Genomic Sciences, [Center for Genomic Sciences, UNAM](https://www.ccg.unam.mx/en/about/)

## Experience {#experience}
* Postdoctoral Research Assistant, [Cancer Systems Biology, Francis Crick Institute](https://www.crick.ac.uk/research/labs/francesca-ciccarelli)
* Postdoctoral Researcher, [Epigenetics, Institute of Biomedicine, University of Gothenburg](https://www.gu.se/en/research/tanmoy-mondal)
* Bioinformatician, [Bioinformatics, Codigo46, R&D](http://linkedin.com/company/codigo46/)
* PhD fellow, [Computational Biology, Institute for Biomedical Research, UNAM](https://www.linkedin.com/in/dr-luis-mendoza)

## Projects {#projects}
### Context-dependent effects of CDKN2A and other 9p21 gene losses during the evolution of oesophageal cancer.
```
   - Analysis of next-generation sequencing (NGS) data in Oesophageal cancer.
   - Multiplexed imaging mass cytometry (IMC) Single-cell analysis.
   - Generated reports and collaborated in writing publications.
```
### Transcriptomics and Epigenetics in Neuroblastoma:
```
    - Developed workflows to analyse epigenomics and transcriptomics data in Neuroblastoma: 
         MeRIP-seq, ChIP-seq, bulk and single-cell RNA-seq and Nanopore long-read sequencing.
    - Coding using R, Shell, and other programming languages. 
    - Implemented workflows in cloud computing and HPC environments: AWS, SLURM.
    - Good coding practices and version control using GitHub.
```

### Bioinformatics at [Codigo46](http://linkedin.com/company/codigo46/): 
```
    * Developed workflows to process and analyse large-scale genetic data for disease risk prediction, pharmacogenomics, and population genetics: 
      Illumina GSA genotyping array, genome-wide association studies (GWAS), polygenic risk score.
    - Led the data mining and annotation team.
    - Conducted statistical analysis, visualisation and interpretation to communicate with stakeholders.
    - Experience working with agile development methodologies (SCRUM).
```
![Analysis of genotypic data at Codigo46](/images/codigo46_workflow.png)

### Dynamical modelling of gene regulatory networks controlling Lymphocyte differentiation
```
   - Network modelling and systems biology. 
   - Developed tools for the dynamical analysis of gene regulatory networks using mathematical modelling.
       
```
![Dynamical modeling of gene regulatory networks controlling Lymphocyte differentiation](/images/network_workflow.png)

## Publications {#publications}
- [(2024). Context-dependent effects of CDKN2A and other 9p21 gene losses during the evolution of oesophageal cancer.](https://www.biorxiv.org/content/10.1101/2024.01.24.576991v1)
- [(2024). METTL3 drives telomere targeting of TERRA lncRNA through m6A-dependent R-loop formation: a therapeutic target for ALT-positive neuroblastoma.]( https://doi.org/10.1093/nar/gkad1242)
-[(2023). Global loss of cellular m6A RNA methylation following infection with different SARS-CoV-2 variants.](https://www.genome.org/cgi/doi/10.1101/gr.276407.12)
- [(2020). Influence of single nucleotide polymorphisms of ELOVL on biomarkers of metabolic alterations in the Mexican population](https://doi.org/10.3390/nu12113389)
- [(2019). Tunicamycin Sensitivity Suppression by High Gene Dosage Reveals New Functions of the Yeast Hog1 MAP Kinase.](https://doi.org/10.3390/cells8070710)
- [(2018). The SQUAD method for the qualitative modelling of regulatory networks.](https://doi.org/10.1007/978-1-4939-8618-7_9)
- [(2016). A Network Model to Describe the Terminal Differentiation of B Cells.](https://doi.org/10.1371/journal.pcbi.1004696)
- [(2015). A dynamical model of the regulatory network controlling lymphopoiesis.](https://doi.org/10.1016/j.biosystems.2015.09.004)
